packages/commonwealth/client/scripts/views/components/react_quill_editor/react_quill_editor.tsx

```diff
@@ -357,6 +403,17 @@ const ReactQuillEditor = ({
           <MarkdownPreview doc={getTextFromDelta(contentDeltaToUse)} />
         )}
       </div>
+      {isModalOpen && (
+        <AddLinkModal
```

Comment:
take a look on other modal implementations.
1. We use CWModal component and pass `open` prop 
2. as a `content` prop we pass a component 

![image](https://github.com/user-attachments/assets/72a8cad9-d648-49c0-903e-60249beff72d)

Reply:
I follow the above example and use the CWModal and pass the content as component

Reply:
While testing the change, I discovered a bug: when the modal is closed, the cursor moves to the beginning of the input field. I’ve also made improvements to address this issue.